trillian/ctfe/config.go

```diff
@@ -118,7 +117,7 @@ func ValidateLogConfig(cfg *configpb.LogConfig) (*ValidatedLogConfig, error) {
 	// Validate the public key.
 	if pubKey := cfg.PublicKey; pubKey != nil {
 		var err error
-		if vCfg.PubKey, err = der.UnmarshalPublicKey(pubKey.Der); err != nil {
+		if vCfg.PubKey, err = x509.ParsePKIXPublicKey(pubKey.Der); err != nil {
```

Comment:
maybe make the error below more golang idiomatic and do:
fmt.Errorf("x509.ParsePKIXPublicKey: %v", err)

Reply:
Done.